src/calculateRentalCost.js

```diff
@@ -4,7 +4,28 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  let basePrice = 0;
+
+  for (let i = 0; i < days; i++) {
+    basePrice += oneDayCost;
+  }
+
+  switch (true) {
+    case days >= longTerm:
+      basePrice -= longTermDiscount;
+      break;
+
+    case days >= shortTerm:
+      basePrice -= shortTermDiscount;
+      break;
```

Comment:
better to use multiple if statements here instead of switch case, because you are not checking some exact value